common/account_utils/src/validator_definitions.rs

```diff
@@ -137,8 +141,7 @@ impl ValidatorDefinitions {
         })?;
         let config_path = validators_dir.as_ref().join(CONFIG_FILENAME);
         if !config_path.exists() {
-            let this = Self::default();
-            this.save(&validators_dir)?;
+            File::create(&validators_dir).map_err(|_e| Error::UnableToCreateValidatorFile)?;
```

Comment:
An empty file is actually invalid for the validator definitions, so I think this should continue using `save` to write a file containing an empty list (it looks like `[]` in the YAML). Rather than `Self::default`, we could add a `Self::create` that creates the lockfile and sets `definitions` to `vec![]`. The lockfile will then be dropped+deleted when `this` goes out of scope.